airbyte-cdk/python/airbyte_cdk/sources/streams/core.py

```diff
@@ -544,4 +544,28 @@ def configured_json_schema(self) -> Optional[Dict[str, Any]]:
 
```

Comment:
@aldogonzalez8 we should implement this filtering at the CDK level so other connectors also benefit from the improvement. Is there a way to move this method's logic to `core.py`?

Reply:
nope. that's my bad. this is absolutely right <3